submissions/lalalilulu/cinema-booking/index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>The first task: movie hall</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8"/>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+<header class="header">
+    <div class="left-header-part">
+        <a href="#"><img class="logo-img" src="assets/cinemax-logo.svg"></a>
+    </div>
+    <div class="right-header-part">
+        <ul>
+            <li><a class="settings-links newsletter" href="#"></a></li>
+            <li><a class="settings-links setting" href="#"></a></li>
+        </ul>
+    </div>
+</header>
+<main class="main">
+    <div class="movie-container">
+        <div class="left-mv-container">
+            <div class="movie-images">
+                <div class="image-card"><img src="assets/tenet_1.jpg" alt=""></div>
+                <div class="image-card"><img src="assets/tenet_2.jpg" alt=""></div>
+                <div class="image-card"><img src="assets/tenet_3.jpg" alt=""></div>
+                <div class="image-card"><img src="assets/tenet_4.jpeg" alt=""></div>
+            </div>
+            <div class="key-info">
+                <span class="info"><b>Премьера: </b>22 августа 2020</span>
```

Comment:
For elements with date/time content apply tag `time`. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)
Also, don't forget to add `datetime` attribute.